src/jose_server.erl

```diff
@@ -718,33 +718,33 @@ rsa_ciphertext(rsa1_5) ->
 	>>;
 rsa_ciphertext(rsa_oaep) ->
 	<<
-		16#B8, 16#F7, 16#0C, 16#A8, 16#F8, 16#30, 16#2A, 16#E9,
-		16#68, 16#8A, 16#DB, 16#3E, 16#5D, 16#AE, 16#84, 16#A7,
-		16#16, 16#FA, 16#9D, 16#E2, 16#FC, 16#81, 16#F7, 16#DF,
-		16#A8, 16#DB, 16#8F, 16#4F, 16#92, 16#A1, 16#51, 16#9E,
-		16#6B, 16#C5, 16#36, 16#CE, 16#93, 16#10, 16#11, 16#D9,
-		16#D5, 16#C2, 16#C9, 16#85, 16#14, 16#EF, 16#D5, 16#C3,
-		16#AC, 16#63, 16#BE, 16#49, 16#FA, 16#02, 16#1A, 16#FC,
-		16#3D, 16#D0, 16#2C, 16#83, 16#C5, 16#76, 16#1D, 16#F5,
-		16#FA, 16#A0, 16#D7, 16#42, 16#ED, 16#3F, 16#A4, 16#12,
-		16#32, 16#14, 16#93, 16#51, 16#79, 16#2E, 16#40, 16#FB,
-		16#14, 16#18, 16#DF, 16#30, 16#62, 16#9F, 16#F3, 16#59,
-		16#5D, 16#83, 16#0F, 16#4A, 16#8F, 16#9B, 16#3F, 16#39
+                16#AF, 16#A4, 16#46, 16#77, 16#B7, 16#39, 16#3A, 16#AB, 
```

Comment:
Demonstration that the ciphertext change here is actually encoded using PKCS#1 Version 1.5 padding:

```elixir
private_key = {:RSAPrivateKey, :"two-prime", 1163626716254914564980136459395684807695125169328499377661515176133500664726004569406515061633752264725167480979816271393911346667208692357841077684025487798285456581385497985022555400834820919818460489376858005605167466539406190291, 65537, 837924159180587806355304330809474848246884082520101181469702392652521428664942454552415662964152625679766070004432092615777291304448520734093291934648299519439752355437698350307876611105690193137070602690197584267484734283766402649, 38384139366830395481042688957876838317068837995135971911011570081580402662762585413480764252529057808049699572389029, 30315300419643147991826027205656337561558478820966564595877651670534732295504135676718770733184467217297653615533079, 8179698344403211976261382028254389488932715739812151665611602419386787670905629917217333011136012044298977741245177, 22006659100424535235227783455286330767217672916176881923934255126503957901011783493597441409146757219020337683430523, 31177545157306276284676329900115896567585136566839161660319287166242766583148091022419866881250561784948462027640129, :asn1_NOVALUE}
public_key = {:RSAPublicKey, 1163626716254914564980136459395684807695125169328499377661515176133500664726004569406515061633752264725167480979816271393911346667208692357841077684025487798285456581385497985022555400834820919818460489376858005605167466539406190291, 65537}
ciphertext = <<175,164,70,119,183,57,58,171,90,12,171,194,173,250,44,147,28,169,89,70,211,10,144,126,155,160,13,180,207,92,166,153,195,113,51,254,49,18,44,194,80,75,133,215,69,78,242,153,69,149,200,186,110,102,121,49,142,183,83,6,32,172,235,182,166,121,159,247,94,137,225,138,129,96,167,121,148,84,239,231,139,51,125,31,229,173,26,102,34,144,42,73,28,163,230,98>>

# PKCS#1 Version 2.2 OAEP-SHA-1 decrypt (this should work)
:error = :jose_jwa_pkcs1.decrypt_private(ciphertext, private_key, [{:rsa_padding, :rsa_pkcs1_oaep_padding}])
:error = :public_key.decrypt_private(ciphertext, private_key, [{:rsa_padding, :rsa_pkcs1_oaep_padding}])

# PKCS#1 Version 1.5 decrypt (this should fail)
"ciphertext" = :jose_jwa_pkcs1.decrypt_private(ciphertext, private_key, [{:rsa_padding, :rsa_pkcs1_padding}])
"ciphertext" = :public_key.decrypt_private(ciphertext, private_key, [{:rsa_padding, :rsa_pkcs1_padding}])
```